cli/ssh.go

```diff
@@ -90,15 +90,27 @@ func (r *RootCmd) ssh() *serpent.Command {
 	wsClient := workspacesdk.New(client)
 	cmd := &serpent.Command{
 		Annotations: workspaceCommand,
-		Use:         "ssh <workspace>",
-		Short:       "Start a shell into a workspace",
+		Use:         "ssh <workspace> [command]",
+		Short:       "Start a shell into a workspace or run a command",
 		Long:        "This command does not have full parity with the standard SSH command. For users who need the full functionality of SSH, create an ssh configuration with `coder config-ssh`.",
```

Comment:
`ssh <workspace> ls -la` works, but `coder ssh <workspace> ls -la` does not. I don't think we have to fix this, but we should have an example on how to pass flags. It is common unix stuff, but worth throwing in here imo.

```suggestion
		Long: "This command does not have full parity with the standard SSH command. For users who need the full functionality of SSH, create an ssh configuration with `coder config-ssh`.\n" +
			FormatExamples(
				Example{
					Description: "Use `--` to separate and pass flags directly to the command executed via SSH.",
					Command:     "coder ssh <workspace> -- ls -la",
				},
			),
```